README.md

```diff
@@ -29,7 +29,7 @@ build.cmd
 
 The `install` task
 will publish the package
-to Sublime Text's *Data* directory.
```

Comment:
>  and restart Sublime Text if it is running.

Would be best to put this on its own line. I won't make you change it, though. :)

Splitting prose lines like this is a trick that was apparently used at Bell Labs and it works well for diffing, etc.